### DISCUSSION TITLE
Loading katex CSS

### DISCUSSION THREAD

**Discussion by AlexanderZeilmann** [timlrx/tailwind-nextjs-starter-blog#386] [Q&A] [answered]
May I ask whether there is a reason to load the CSS file of katex from jsdelivr instead of the `node_modules` folder? I changed this on my website and everything works fine. After this change we can also drop jsdelivr from the CSP.

**Comment by timlrx** [comment, marked as answer]
I didn't realize that they included the CSS file in the installed package and just blindly followed their installation instructions. Will make the change, thanks!
